File: Day-16/Pattern23.py
n = int(input())
def PatternTwentyThree(n):
    i = 1
    while(i<=n):
        space = n-i
        while(space):
            print(' ', end='')
            space -= 1
        
        nums = i
        while(nums):
            print(i, end='')
            nums -= 1
        
        i+=1
        print(end='\n')

PatternTwentyThree(n)

File: Day-17/Pattern23.js
function PatternTwentyThree(n) {
  let i = 1;
  while (i <= n) {
    let str = "";
    let spaces = n - i;
    while (spaces) {
      str += " ";
      spaces--;
    }

    let nums = i;
    while (nums) {
      str += i;
      nums--;
    }

    console.log(str);
    i++;
  }
}

PatternTwentyThree(4);
